config/connect.js

```diff
@@ -1,10 +1,24 @@
 /**
- * The different Call To Action buttons on the Connect tab
+ * The different items in the Connect tab
  */
 
 import Colors from '../constants/Colors';
 
-export default [
+/**
+ * The main Connect items
+ */
+export const listItems = [
```

Comment:
this is config for the list items 🔥 
we can easliy delete items, but since we need to have a native solution for these pages, it requires dev work to add a new page